spec.html

```diff
@@ -3117,89 +3117,21 @@
     <!-- es6num="7.1.1" -->
     <emu-clause id="sec-toprimitive" aoid="ToPrimitive">
       <h1>ToPrimitive ( _input_ [ , _PreferredType_ ] )</h1>
-      <p>The abstract operation ToPrimitive takes an _input_ argument and an optional argument _PreferredType_. The abstract operation ToPrimitive converts its _input_ argument to a non-Object type. If an object is capable of converting to more than one primitive type, it may use the optional hint _PreferredType_ to favour that type. Conversion occurs according to <emu-xref href="#table-9"></emu-xref>:</p>
-      <emu-table id="table-9" caption="ToPrimitive Conversions">
```

Comment:
I added a link here to preserve this id (seems plausible there are inbound references to this table).